pkg/operator/controller/ingress/deployment_test.go

```diff
@@ -79,6 +79,9 @@ func checkRouterContainerSecurityContext(t *testing.T, deployment *appsv1.Deploy
 			if v := container.SecurityContext.AllowPrivilegeEscalation; v == nil || *v != true {
 				t.Errorf("%s container does not have securityContext.allowPrivilegeEscalation: true", routerContainerName)
 			}
+			if v := container.SecurityContext.ReadOnlyRootFilesystem; v == nil || *v != false {
```

Comment:
```suggestion
			if v := container.SecurityContext.ReadOnlyRootFilesystem; v == nil || !*v {
```

Reply:
You got it backwards, it should be `... || *v` (which probably proves the point that the condition is written poorly), but I get what you're aiming at.  I've made the appropriate change.

Reply:
Touché. Something other than `v` would also be more informative.

Reply:
True. I've updated the variables to be more informative in my latest push.